src/truffle-integration/projectsWatcher.js

```diff
@@ -121,7 +122,15 @@ class ProjectsWatcher extends EventEmitter {
           this.subscribedTopics.indexOf(entry.signature) === -1
         );
       });
-      topics = topics.concat(abiEvents.map(event => event.signature));
+      topics = topics.concat(
+        abiEvents.map(abi => {
```

Comment:
LGTM!  Although maybe it's a little verbose?  Thoughts on a ternary or short-circuit eval?